make/run

```diff
@@ -26,16 +26,33 @@ done
 kubectl create namespace "${NAMESPACE}"
 stampy "${GIT_ROOT}/scf_metrics.csv" "$0" make-run::namespace end
 
-stampy "${GIT_ROOT}/scf_metrics.csv" "$0" make-run::ca-cert start
+helm_args=(
+    --name "${NAMESPACE}"
+    --namespace "${NAMESPACE}"
+    --values "bin/settings.yaml"
+    --set "kube.storage_class.persistent=${STORAGE_CLASS}"
+)
 
-printf "Waiting for uaa cert to be generated by the uaa chart...\n"
-until has_internal_ca ; do
-    sleep 1
-done
+if [ "${EMBEDDED_UAA:-false}" != "true" ]; then
```

Comment:
It would make more sense to have `EMBEDDED_UAA` in the https://github.com/SUSE/scf/blob/develop/make/include/defaults.

Reply:
I don't know, we have plenty of tests like this in the scripts:

```
if [ -z "${INGRESS_CONTROLLER:-}" ]; then
```

And this is just the same thing: checking if the variable has been set or not. Splitting things up into multiple files does not make the code easier to understand.